### PR TITLE
Ambassador_wahl1-post

### PR DESCRIPTION
**Pull request by crazy4pi314**
@misty-w I think this is ready to go! Have a look and see if it matches what  you drafted before

### CODE REVIEW

**Comment by Misty-W** [comment]
LGTM @crazy4pi314, looking forward to seeing it on the website!

**Review by nathanshammah — APPROVED** [review]
I reviewed the markdown and it reads very well, well done @Misty-W!

I did not check that the image links are working.

The file name looks a bit strange. I imagine the "1" is to order w.r.t. other ambassador stories. I propose to change it to "ambassador_1_wahl" and keep that convention, so that they are also ordered when listed in a folder.

File: posts/_markdown/ambassador_wahl1.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+The challenge presented in the hackathon was to create a tutorial example demonstrating the application of one of the error mitigation techniques in Mitiq to a variational problem. Without error mitigation, noise from the quantum computer would result in poor convergence or non-convergence of the algorithm. Adding error mitigation would then allow convergence or improve convergence. **Part of what interested me about the problem was that it was more research-oriented**, since there was no guarantee that the desired behavior was actually achievable, although previous demonstrations indicated a solution was likely to exist.
+
+![Github issue from unitaryHACK](..\images\ambassador_wahl1_card.png "Github issue from unitaryHACK")
```
Comment (nathanshammah) [inline posts/_markdown/ambassador_wahl1.md:14]:
```suggestion
![Github issue from unitaryHACK](../images/ambassador_wahl1_card.png "Github issue from unitaryHACK")
```

File: posts/_markdown/ambassador_wahl1.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+**While investigating how to make the optimization efficient on longer circuits, I learned that pyQuil supports parametric compilation**. For parametric programs, the compilation is done outside the optimization loop, and the parameters are updated via a `memory map` (in v2) or the method `program.write_memory` (in v3). When I attempted to pass a parametric program to Mitiq and apply ZNE, **I discovered that Mitiq would not accept the memory map. I filed an issue to that effect, and it was fixed within a few days!** Later I found that support for parametric programs was not entirely necessary for the VQE example, once the compilation step was removed to allow use of custom gates. Instead, I repurposed the demonstration of [ZNE with pyQuil and parametric compilation](https://mitiq.readthedocs.io/en/stable/examples/pyquil_demo.html) as a separate tutorial.
+
+![Energy landscape plot from “ZNE with pyQuil and parametric compilation” tutorial](..\images\ambassador_wahl1_pyquil.png "Energy landscape plot from “ZNE with pyQuil and parametric compilation” tutorial")
```
Comment (nathanshammah) [inline posts/_markdown/ambassador_wahl1.md:30]:
```suggestion
![Energy landscape plot from “ZNE with pyQuil and parametric compilation” tutorial](../images/ambassador_wahl1_pyquil.png "Energy landscape plot from “ZNE with pyQuil and parametric compilation” tutorial")
```

File: posts/_markdown/ambassador_wahl1.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+In the end, once I defined an appropriate circuit and noise model, I was able to show that applying ZNE enhanced convergence to the minimum energy. While the VQE algorithm is generally considered to be robust to noise (see the [VQE tutorial in pyQuil / Grove](https://grove-docs.readthedocs.io/en/latest/vqe.html)), I observed that without mitigation, the accumulation of errors still resulted in loss of accuracy and additional iterations required to reach convergence. While there was some variation from run to run, applying ZNE generally improved convergence to the minimum energy in terms of accuracy, number of iterations, or both. The performance increase from ZNE was most evident in the energy landscape plot, where the noisy landscape was noticeably flatter than the landscape generated with ZNE.
+
+![Energy landscape plot from “Variational Quantum Eigensolver improved with Zero Noise Extrapolation” tutorial](..\images\ambassador_wahl1_landscape.png "Energy landscape plot from “Variational Quantum Eigensolver improved with Zero Noise Extrapolation” tutorial")
```
Comment (nathanshammah) [inline posts/_markdown/ambassador_wahl1.md:36]:
```suggestion
![Energy landscape plot from “Variational Quantum Eigensolver improved with Zero Noise Extrapolation” tutorial](../images/ambassador_wahl1_landscape.png "Energy landscape plot from “Variational Quantum Eigensolver improved with Zero Noise Extrapolation” tutorial")
```

**Review by nathanshammah — CHANGES_REQUESTED** [review]
I made some changes based on the formatting in [the Krylov blogpost](https://github.com/unitaryfund/unitary-fund/blob/4de949b7b341b1bfa9e621807c2db4d4ef67b6c8/posts/_markdown/krylov.md), where from the markdown file the figures show up. So I proposed these changes to mimic that. 

I would also change the title of the files (previous comment).

**Comment by nathanshammah** [comment]
Looking at #155, there the path there is `.../.../images`, not just `../images`. The first path is correct to visualize the images from a md file since there is a  `posts/_markdown` folder, which is one level lower than the one for html files, `posts`. Looking at [VQA post raw file](https://raw.githubusercontent.com/unitaryfund/unitary-fund/master/posts/_markdown/mitiq_vqa_performance.md), it looks like the correct one is `../` only. It may be nice to visualize both posts (also in markdown), either by placing at the same level from the image folder, or specifying this in someway that works for both.

**Comment by crazy4pi314** [comment]
> I reviewed the markdown and it reads very well, well done @Misty-W!
> 
> I did not check that the image links are working.
> 
> The file name looks a bit strange. I imagine the "1" is to order w.r.t. other ambassador stories. I propose to change it to "ambassador_1_wahl" and keep that convention, so that they are also ordered when listed in a folder.

The number here was because some ambassadors may write more than one post. I will change the number to intro, to avoid confusion.

**Comment by crazy4pi314** [comment]
Ok, I have fixed all the path issues, the one thing I noticed was that it's missing some links for where folks can find you @Misty-W! If you want to comment or PR what links/email/handles you want (no obligation to any in particular) then I can get this merged!

**Comment by Misty-W** [comment]
Twitter: @mistyawahl
Github: https://github.com/Misty-W
LinkedIn: https://www.linkedin.com/in/misty-wahl

**Comment by crazy4pi314** [comment]
> @crazy4pi314, that reminds me, I also submitted a bio awhile back, will that be published as well?
 Yup, I am working on getting a people page with ambassadors on the website :)

**Pull request merged**